File: TestLabelButton/TestLabelButton/ViewController.m
//
//  ViewController.m
//  TestLabelButton
//
//  Created by Podyplomowe on 10.01.2015.
//  Copyright (c) 2015 Podyplomowe. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}




- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)button1Pressed:(UIButton *)sender {
     self.label.text =  sender.titleLabel.text;
}


- (IBAction)button2Pressed:(UIButton *)sender {
    [[self label] setText:[[sender titleLabel] text]];
}

- (IBAction)digitPressed:(UIButton *)sender {
    
    //    [[self display] setText:[[sender titleLabel] text]];
    
    self.label.text =  sender.titleLabel.text;
    
}

@end

File: 2015.03.01_Swift2_Cars/Cars1/ViewController.swift
//
//  ViewController.swift
//  Cars1
//
//  Created by IIUJ on 01.03.2015.
//  Copyright (c) 2015 IIUJ. All rights reserved.
//

import UIKit

class ViewController: UIViewController,UITextFieldDelegate {

    @IBOutlet weak var producerTextField: UITextField!
    
    @IBOutlet weak var modelTextField: UITextField!
    
    @IBOutlet weak var numberOfCarsLabel: UILabel!
    
    var cars = [Car]()
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        producerTextField.delegate = self
        modelTextField.delegate = self
    }

    @IBAction func backgroundTouched(sender: UIControl) {
        self.view.endEditing(true)
    }
    
    @IBAction func addCar() {
        
        cars.append(Car(producer: producerTextField.text, model: modelTextField.text))
        
        numberOfCarsLabel.text = String(cars.count)
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    func textFieldShouldReturn(textField: UITextField) -> Bool {
        textField.resignFirstResponder()
        return true
    }

}

